lib/fs_utils/file_list.js

```diff
@@ -163,6 +164,7 @@ class FileList extends EventEmitter {
   copy(asset) {
     const path = asset.path;
     const resetCopy = (p) => {
+      if (this.disposed) { return p; }
```

Comment:
It appears to have been caused by first removing the temp dir and only *then* closing the watcher. https://github.com/brunch/brunch/pull/1370 deals with that underlying cause.